### PR TITLE
added example

### PR DESCRIPTION
**Pull request by eppinger**

### CODE REVIEW

File: data/en/cfmailpart.json [hunk 1]
```diff
@@ -15,5 +15,12 @@
 		"railo": {"minimum_version":"", "notes":"", "docs":"http://railodocs.org/index.cfm/tag/cfmailpart"},
 		"openbd": {"minimum_version":"", "notes":"", "docs":"http://openbd.org/manual/?/tag/cfmailpart"}
 	},
-	"links": []
+	"links": [],
+	"examples": [
+		{
+		"title": "Script syntax using function call",
+		"description": ".",
+		"code": "cfmail( to = \"recipient@example.com\", from = \"sender@example.com\", subject = \"Example email\" ) {\ncfmailpart(type=\"text/plain\") {\nwriteOutput(\"If you are seeing this, your e-mail client does not support HTML messages.\");\n}\ncfmailpart(type=\"text/html\") {\nwriteOutput(htmlVersionOfMessage);\n}\n}"
```
Comment (shaedrich) [inline data/en/cfmailpart.json]: Could you add some indention?
```suggestion
		"code": "cfmail( to = \"recipient@example.com\", from = \"sender@example.com\", subject = \"Example email\" ) {\n\tcfmailpart(type=\"text/plain\") {\n\t\twriteOutput(\"If you are seeing this, your e-mail client does not support HTML messages.\");\n\t}\n\tcfmailpart(type=\"text/html\") {\n\t\twriteOutput(htmlVersionOfMessage);\n\t}\n}"
```

Comment (eppinger) [reply data/en/cfmailpart.json]: i'm nor sure what you mean ?

The indention for the second commit ?

Comment (shaedrich) [reply data/en/cfmailpart.json]: Why did you delete the tabs?

Comment (eppinger) [reply data/en/cfmailpart.json]: because travis failed and i thought, this was the reason

Comment (shaedrich) [reply data/en/cfmailpart.json]: I don't think, it fails because of tabs. Do you have the error message for chance?

Comment (eppinger) [reply data/en/cfmailpart.json]: no, only the result from travis 
https://travis-ci.org/foundeo/cfdocs/jobs/482437644

should i readd the tabs and try again ?

**Comment by pfreitag** [comment]
Thanks for the PR @eppinger

**Pull request merged**